components/map/map.jsx

```diff
@@ -10,61 +16,146 @@ const POKE_MARKERS = [
   'https://res.cloudinary.com/joycechau/image/upload/v1487572666/premierball.png'
 ];
 
-const MAX_LAT = 49.5;
-const MIN_LAT = 17.5;
-const MAX_LNG = 294.5;
-const MIN_LNG = 236;
+const MAX_LAT = 85;
+const MIN_LAT = -85;
+const MAX_LNG = 180;
+const MIN_LNG = -180;
 
 export default class Map extends React.Component {
   constructor(props) {
     super(props);
+    this.state = { map: null, pokemon: null, modalOpen: false };
+    this.openModal = this.openModal.bind(this);
+    this.closeModal = this.closeModal.bind(this);
   }
 
   componentDidMount() {
     const map = new google.maps.Map(document.getElementById('map'), {
       zoom: 4,
-      maxZoom: 4,
       minZoom: 4,
-      center: {lat: 35, lng: 265},
-      zoomControl: false,
+      center: { lat: 35, lng: -95 },
+      zoomControl: true,
       mapTypeControl: false,
       scaleControl: false,
       streetViewControl: false,
       rotateControl: false,
       fullscreenControl: false,
-      panControl: false,
-      draggable: false,
-      scrollwheel: false,
+      panControl: true,
+      draggable: true,
+      scrollwheel: true,
+    });
+    const infoMarker = new google.maps.Marker({
+      position: { lat: 35, lng: -95 },
+      icon: POKEBALL_MARKERS[4],
+      map: map
+    });
+    const infoWindow = new google.maps.InfoWindow({
+      content: "<div style='width: 212px; font-size: 10px; margin-top: 5px;'>" +
+                 "Click on pokeballs to search for pokemon!" +
+               "</div>"
+    });
+
+    infoWindow.open(map, infoMarker);
+
+    infoMarker.addListener('click', () => {
+      infoMarker.setMap(null);
+      this.addPokeballMarker(map);
+
+      const markerLat = infoMarker.getPosition().lat();
+      const markerLng = infoMarker.getPosition().lng();
+
+      map.panTo({ lat: markerLat, lng: markerLng });
+      map.setZoom(10);
+
+      this.addPokemonMarker(map, markerLat, markerLng);
     });
 
-    for (var i = 0; i < 25; i++) {
-      this.addMarker(map);
+    for (let i = 0; i < 750; i++) {
```

Comment:
Usually when you use some number it's better to make it a const similar to the MAX_LAT etc. So this const should be TOTAL_POKEBALLS or something